tests/server/routes/orders-router-test.js

```diff
@@ -16,41 +18,37 @@ describe('API Orders Routes', function () {
   beforeEach('Create app', function () {
     app = require('../../../server/app')(db)
     Order = db.model('order')
-    agent = supertest.agent(app)
+    agent = promisedSupertest.agent(app)
   })
 
   beforeEach(function () {
     return Promise.all([
       Order.create({}),
-      Order.create({ status: 'completed' }),
-      Order.create({ status: 'completed' }),
+      Order.create({ status: completed }),
+      Order.create({ status: completed }),
```

Comment:
Just curious - why doesn't the string version work?  doesn't the order model take in status as strings? 
status: {
      type: Sequelize.ENUM('created', 'processing', 'cancelled', 'completed'),
      defaultValue: 'created'
}